functions/consumer/rsocket-consumer/src/main/java/org/springframework/cloud/fn/consumer/rsocket/RsocketConsumerProperties.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2020-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.fn.consumer.rsocket;
+
+import java.net.URI;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties("rsocket.consumer")
+public class RsocketConsumerProperties {
+
+	/**
+	 * RSocket host.
+	 */
+	private String host = "localhost";
+
+	/**
+	 * RSocket port.
+	 */
+	private int port = 7000;
+
+	/**
+	 * URI that can be used for websocket based transport.
+	 */
+	private URI uri;
+
+	/**
+	 * Route used for RSocket.
+	 */
+	private String route;
+
+	public String getHost() {
+		return host;
```

Comment:
No `this.`?

What kind of Checkstyle config do we use? 
Does it really let us to skip `this.`?